build_library/sbsign_util.sh

```diff
@@ -0,0 +1,55 @@
+# Copyright (c) 2024 The Flatcar Maintainers.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+if [[ ${COREOS_OFFICIAL:-0} -ne 1 ]]; then
+    SBSIGN_KEY="/usr/share/sb_keys/shim.key"
+    SBSIGN_CERT="/usr/share/sb_keys/shim.pem"
+else
+    SBSIGN_KEY="pkcs11:token=flatcar-dev-cert"
+    unset SBSIGN_CERT
+fi
+
+PKCS11_MODULE_PATH="/usr/$(get_sdk_libdir)/pkcs11/azure_kms_pkcs11.so"
+
+PKCS11_ENV=(
+    AZURE_KEYVAULT_URL="https://chewi-test.vault.azure.net/"
+    PKCS11_MODULE_PATH="${PKCS11_MODULE_PATH}"
+    AWS_KMS_PKCS11_DEBUG=1
+)
+
+get_sbsign_cert() {
+    if [[ ${SBSIGN_KEY} != pkcs11:* || -s ${SBSIGN_CERT} ]]; then
```

Comment:
I think this will fail with `set -euo pipefail` for unofficial builds, because SBSIGN_CERT is undefined:

```
$ set -euo pipefail
$ SBSIGN_KEY="pkcs11:token=flatcar-dev-cert"
$ unset SBSIGN_CERT
$ if [[ ${SBSIGN_KEY} != pkcs11:* || -s ${SBSIGN_CERT} ]]; then echo 'early return'; else echo 'get the cert'; fi
bash: SBSIGN_CERT: unbound variable
```

So maybe:
```suggestion
    if [[ ${SBSIGN_KEY} != pkcs11:* || -s ${SBSIGN_CERT-} ]]; then
```